user/how-to-guides/how-to-reinstall-a-template.md

```diff
@@ -60,27 +59,27 @@ If you want to reinstall more than one template, repeat these instructions for e
 3. Uninstall the target template from dom0:
 
     ```
-    $ sudo dnf remove <template-package-name>
+    $ qvm-template remove <template-name>
     ```
 
-   For example, to uninstall the `whonix-gw` template:
+   For example, to uninstall the `whonix-gw-16` template:
 
     ```
-    $ sudo dnf remove qubes-template-whonix-gw
+    $ qvm-template remove whonix-gw-16
     ```
 
 4. Reinstall the target template in dom0:
 
     ```shell_session
-    $ sudo qubes-dom0-update --enablerepo=<optional-additional-repo> \
-       <template-package-name>
+    $ qvm-template  --enablerepo=<optional-additional-repo> \
+       <template-name>
     ```
 
-   For example, to install the `whonix-gw` template:
+   For example, to install the `whonix-gw-16` template:
 
     ```shell_session
-    $ sudo qubes-dom0-update --enablerepo=qubes-templates-community \
-       qubes-template-whonix-gw
+    $ qvm-template install --enablerepo=qubes-templates-community \
```

Comment:
```suggestion
    $ qvm-template --enablerepo=qubes-templates-community install \
```
the document should stay consistent, the subcommand is given after the parameters, and this matches the command's synopsis in the man page https://dev.qubes-os.org/projects/core-admin-client/en/latest/manpages/qvm-template.html#synopsis